airflow/dags/rt_loader/external_validation_service_alerts.sql

```diff
@@ -1,38 +1,11 @@
 ---
 operator: operators.SqlQueryOperator
-description: |
-  GTFS RT validation errors as returned by the validator. Each entry corresponds
-  to a <filename>.results.json file that is spit out by the validator.
-fields:
-  errorMessage: |
-    An object with fields for messageId, gtfsRTFeedIterationModel, validationRule,
-    errorDetails.
-  occurrenceList: |
-    An array of objects with these fields: occurenceId, messageLogModel, prefix
+dependencies:
+    - load_rt_validations
```

Comment:
Can we re-add field documentation for these?

Reply:
Can do.